temping.gemspec

```diff
@@ -15,6 +15,8 @@ Gem::Specification.new do |s|
     s.add_development_dependency "activerecord-jdbcsqlite3-adapter", "~> 1.2.9"
   else
     s.add_development_dependency "sqlite3", "~> 1.3.10"
+    s.add_development_dependency "pg", "~> 0.18.2"
+    s.add_development_dependency "mysql", "~> 2.9.1"
```

Comment:
Seems like [mysql2](https://github.com/brianmario/mysql2) is a much more popular choice in the Rails world. 
Do you think it would be much hassle to change it to use that one?